README.md

```diff
@@ -7,19 +7,18 @@
 
 ## Requirements
 
-* nodejs 10 LTS
+* nodejs 10 or 12 LTS
```

Comment:
I generally don't support node versions beyond what screeps currently supports, in this case, v10. While it _may_ work on newer versions, officially screeps only supports 10, and that means I only officially support 10 for my mods. Too many random other issues to do otherwise.

Reply:
The readme at https://github.com/screeps/screeps claims "Node.js 10 LTS or higher" (the requirements on that readme are somewhat outdated though).  Although Node 14 did not work for me, I have not had any problems with 12.  If you consider 10 to be the only version that is actually supported, then I can remove that change.

Reply:
🤔 Hmm, maybe should just use the same terminology here, Nodejs 10 LTS or higher, or just remove the line entirely, since screeps requires it anyway. The mod itself technically doesn't care as long as its not older node versions than whatever JS features I use. Its just screeps and ivm/pathfinder that tend to limit versions.